### PR TITLE
Standard Readme file

### PR DESCRIPTION
**Pull request by hcarreras**
The goal of this README file is to be used as a template for new projects and inspiration for existing README files. 

The intention of the README is not to act as a hard guideline on what should or should not go into a README. Instead, it offers a space to include the most relevant information. It is the responsibility of the developers of each project to decide what is the most pertinent information and modify this template to accommodate the needs.

### CODE REVIEW

File: standard_readme.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Name of the project
+
+Description of the project. What is it solving? Who are the users?
+
+URL to the project
```
Comment (bohme) [inline standard_readme.md]: I'd make this URL**S** given the staging/production setup we generally use.

File: standard_readme.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## Documentation
+
+Where to find the documentation.
+
+# Production
+
+## Automatic deploy
+
+How does automatic deploy happen?
```
Comment (bohme) [inline standard_readme.md]: 'How do automatic deploys happen?' sounds a little... I dunno softer? it rolls better of the tongue I'd say 🤔

**Comment by heatherm** [comment]
Also, a section for Notable inclusions & Notable exclusions are always handy.

File: standard_readme.md (hunk 1 above)
Comment (GeoffAbtion) [reply standard_readme.md]: URLS to related github repositories for when there are frontend/backend repos.

**Comment by hcarreras** [comment]
I've implemented the suggestions 💪

File: standard_readme.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+- [Name of the project](#name-of-the-project)
+- [Development](#development)
+  * [Requirements](#requirements)
+  * [Set up the app the first time](#set-up-the-app-the-first-time)
+  * [Run the app](#run-the-app)
+  * [Run the tests](#run-the-tests)
+  * [Useful commands](#useful-commands)
+  * [Download production or staging DB](#download-production-or-staging-db)
+  * [Workflow](#workflow)
+  * [Documentation](#documentation)
```
Comment (GeoffAbtion) [inline standard_readme.md]: This looks like a Development documentation section, where I might put links to Rspec or Rubocop documentation.

There needs to be a place for links to things like API documentation for *this* project.
Maybe that belongs under Production? or in it's own top level section?

**Comment by hcarreras** [comment]
Example following this guideline: https://github.com/abtion/fact/pull/1036

File: standard_readme.md (hunk 3 above)
Comment (hcarreras) [reply standard_readme.md]: The idea was to put here things like the API documentation for _this project_. You might be right that Development might not be the right category. Perhaps something we should put at the top?

I would place `Rspec` or `Rubocop` documentations under `#notable-inclusions-and-notable-exclusions`

**Pull request merged**